.envrc

```diff
@@ -0,0 +1,27 @@
+# This file is used by direnv to setup the environment when entering to use fvm's flutter.
```

Comment:
Please do keep prose to around 68 columns wide, and at most 70. It makes a real difference in readability — for example here's what the commit message looks like in my terminal:
![image](https://github.com/zulip/zulip-flutter/assets/28173/7d4a1c11-c5bd-42cb-8ea3-6ec90de212c4)

Even when the terminal is wide and the lines don't wrap, prose wider than around 70 columns gets less comfortable to read as your eyes have to scan back and forth across the lines.

This guideline is explicit in our commit style guide:
https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html#formatting-guidelines
but it applies to prose elsewhere, like Markdown and comments, too.